ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.8.yaml

```diff
@@ -0,0 +1,188 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  stolostron_builder_go1.18-linux:
```

Comment:
Can we add go1.19 image as well? We should upgrade to 1.19 in global-hub side.

Reply:
done.